docs/evm_compatibility/code_compatibility_checklist.md

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Smart Contract Code Compatibility Checklist'
+proofedDate: na
+iterationBy: HB
+includedInSite: false
+approvedBy: na
+comments: na
+---
+
+## Introduction
+
+This page details about the smart contract code compatibility factors that determines whether the smart contracts are fully compatible with Neon EVM or not.
+
+## Smart contract compatibility factors
+
+The following are the factors that determine the smart contract code compatibility with Neon EVM and the alternative solutions -
+
+### Solidity Compiler version
+
+If the contracts are not clean fork like Uniswap V2 or Aave V2 and the solidity compiler is very old i.e, < 0.8.x, it is always recommended to upgrade to latest stable solidity compiler version and re-run the tests.
+:::info
+Solidity version ≥ 0.8.25 is not supported currently because of the need to implement EIP-5656 and EIP-1153. It will be supported on Devnet and Mainnet soon.
+:::
+
+### Usage of third party protocols
+
+There shouldn't be any usage of third party protocols that are not currently supported on Neon EVM.
```

Comment:
This will basically include the [ecosystem page](https://neonevm.org/ecosystem) where all the 3rd party protocols supported are listed. For example, 
1. Indexers: The Graph
2. Multisigs: Safe(fork)
3. Oracles: Chainlink, Pyth

Reply:
@OlehSyzonov Should I include the ecosystem page link?

Reply:
Thank you for the explanation. I think, probably not time. Right now it doesn't seem be comprehensive source. For example, it doesn't contain _The Graph_.

Reply:
Okay, @OlehSyzonov. Then we can add the ecosystem page later on when it is completely updated.